runtime/libc/wasi/wasi_impl_minimal.c

```diff
@@ -0,0 +1,1129 @@
+#include <assert.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <limits.h>
+#include <math.h>
+#include <printf.h>
+#include <setjmp.h>
+#include <signal.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+#include <unistd.h>
+
+#include <sys/stat.h>
+#include <sys/uio.h>
+
+#include "wasi_impl.h"
+
+/*
+ * Essentially based off the uvwasi_t
+ * TODO: Add file descriptors in support of preopens
+ */
+typedef struct wasi_context_s {
+    __wasi_size_t argc;
```

Comment:
I'd expect some of the previous comments to be blown off since they were to imported code from node. So I'll repeat here: validate that is the 4 space indents are typical in awsm.

Reply:
All of the derived code from Node.js/uvwasi has downstream changes and has been run through clang-format using the local project config.